go-controller/pkg/ovn/secondary_layer3_network_controller_test.go

```diff
@@ -197,15 +197,15 @@ var _ = Describe("OVN Multi-Homed pod operations", func() {
 			dummyPrimaryUserDefinedNetwork("192.168.0.0/16"),
 			nonICClusterTestConfiguration(),
 		),
-		table.Entry("pod on a user defined secondary network on an interconnect cluster",
+		table.Entry("pod on a user defined secondary network on an IC cluster",
 			dummySecondaryUserDefinedNetwork("192.168.0.0/16"),
 			icClusterTestConfiguration(),
 		),
-		table.Entry("pod on a user defined primary network on an interconnect cluster",
+		table.Entry("pod on a user defined primary network on an IC cluster",
 			dummyPrimaryUserDefinedNetwork("192.168.0.0/16"),
 			icClusterTestConfiguration(),
 		),
-		table.Entry("pod on a user defined primary network on an interconnect cluster",
+		table.Entry("pod on a user defined primary network on an IC cluster with per-pod SNATs disabled",
```

Comment:
ditto

Reply:
Some confusion here because the change of interconnect to IC is already on the second commit so you might have made this suggestion from the second commit.

Reply:
no my suggestion is to make it per-pod SNAT enabled :D

Reply:
my comment didn't have anything to do with IC...

Reply:
I am referring to the fact that in your suggestion you changed `interconnect` for `IC`, literally, when that is already part of my changes, so it was confusing.